### ISSUE TITLE
Using LEOPARD for other PGS methods, and linearly combining SNP-weights

### ISSUE THREAD

**Issue by opain** [qlu-lab/X-Wing#5]
Hello again,

I would like to use LEOPARD to estimate the optimal linear combination of PGS derived using other PGS methods. For example, if I apply ldpred2 separately to two GWAS from different populations, I would like to use LEOPARD to estimate the optimal combination of the PGS in given target population. I would appreciate your advice on how to best achieve this.

1. Can LEOPARD be applied to PGS other than the output of PANTHER? I assume so, since the method seems similar to that used by PUMAS-ensemble, which can apparently be applied to any PGS method.
2. How does LEOPARD compare SNP-weights that may not be on the same scale? Some methods standardise the effect sizes, while others do not necessarily (such as pT+clump when using non-standardised GWAS effect sizes). Should I be scaling the effect sizes in advance of using LEOPARD?
3. This questions applies to when using X-Wing as well. You suggest standardising the two PGS in the target population, before linearly combining using the weights from LEOPARD. I would like to produce SNP-weights that reflect that linear combination, so I am scaling the SNP-weights for the PGS, so that the SD of the resulting PGS equals 1. I assume this achieves the same thing? I am calculating the PGS in some target reference data, and multiplying the SNP-weights by the 1/PGS SD.

Any other cautionary advice is welcome!

Many thanks,

Ollie

**Comment by jmiao24** [comment]
Hi Ollie,

1. Yes, it can be applied to any PGS, not just the output of PANTHER.
2. Since LEOPARD is linearly combining PGS (instead of SNP weights), different scaling of the SNP weights should not matter too much. As long as the scale of SNP weights is consistent with the scale of SNP (standardized or allele scale), it should work fine.
3. Yes, scaling all SNP weights by the same factor should achieve the same result.

Best,
Jiacheng

**Comment by opain** [comment]
Thank you for your reply Jiacheng. This is really useful. I am applying LEOPARD to a few different PGS methods. It appears the performance of LEOPARD varies across PGS methods, and wanted to check that I was not misusing the LEOPARD approach. Perhaps LEOPARD performance varies across PGS methods due to their different sensitivity to the size of the LD reference, which is quite small when using the subsets of 1KG data. I will continue with my analysis, and let you know if I have any other questions.

Many thanks,

Ollie

**Comment by opain** [comment]
Apologies for reopening this issue. I wanted to clarify my third question - regarding scaling the PGS. Looking back at your instructions, I realised that you recommend 'centering' the PGS  before applying mixing weights, rather than 'scaled', as I had asked before. Can I confirm that the PGS should be centered (i.e. PGS - mean(PGS)), but the scale of the PGS should not be adjusted?

Many thanks,

Ollie

**Comment by jmiao24** [comment]
Hi Ollie,

Either centering or scaling should work fine. The key is to ensure that the intercept in the linear regression used to obtain the combination weights is set to zero.

Best,
Jiacheng

**Comment by opain** [comment]
Hi Jiacheng,

Thank you for your reply. I’ll proceed with centering the PGS only. Would altering the scale (e.g., setting the SD to 1) for the population-specific PGS risk making them inconsistent with the mixing weights from LEOPARD, given that the reference PGS are not scaled in the LEOPARD analysis?

Sorry if I am confused.

Many thanks,

Ollie

**Comment by jmiao24** [comment]
Yes, I would recommend using the same scaling approach (either scaled or unscaled) for both the population-specific PGS and the reference PGS to ensure consistency in the analysis.

Best,
Jiacheng

**Comment by opain** [comment]
OK sounds good. Thank you Jiacheng
